be/src/common/config.h

```diff
@@ -1369,6 +1369,13 @@ CONF_String(rocksdb_cf_options_string, "block_based_table_factory={block_cache={
 
 CONF_String(rocksdb_db_options_string, "create_if_missing=true;create_missing_column_families=true");
 
+// It is the memory percent of write buffer for meta in rocksdb.
+// default is 5% of system memory.
+CONF_mInt64(rocksdb_write_buffer_memory_percent, "5");
+// It is the max size of the write buffer for meta in rocksdb.
+// default is 1GB.
+CONF_mInt64(rocksdb_max_write_buffer_memory_bytes, "1073741824");
```

Comment:
Also update the BE_Configuration.md doc as well.

Reply:
done